### DISCUSSION TITLE
How to setup Biome and prettier depending on project

### DISCUSSION THREAD

**Discussion by e-osuna-g** [LazyVim/LazyVim#2289] [Q&A]
I want to be able to set up formatting/linting depending on if the project has a biome.json or prettier, but every time I install biome trough mason, it lints the files even tho there's no biome.json in the project.

**Comment by madflow** [comment]
Maybe restricting the `root_pattern` to just "biome.json" will work: https://github.com/neovim/nvim-lspconfig/blob/master/doc/server_configurations.md#biome ?

**Reply by e-osuna-g** [reply]
oh lspconfig is a plugin right?, i'm not sure i have it. but i'll try with that

edit: would this help with both formatting and linting?
